modules/core/src/main/java/org/apache/fluo/core/impl/TransactionImpl.java

```diff
@@ -27,12 +30,6 @@
 import java.util.Objects;
 import java.util.Set;
 
-import com.google.common.annotations.VisibleForTesting;
```

Comment:
The build is failing because checkstyle doesn't like this import order.  We have instructions for ordering imports in  Eclipes and IntelliJ at : 

http://fluo.apache.org/how-to-contribute/#coding-guidelines